src/configs/default.json

```
@@ -31,11 +31,14 @@
             "rankCacheAge": 86400000,
             "levelCaps": [
                 40,
-                41,
                 50,
                 51
             ],
+            "v1": true,
+            "v2": true,
+            "v2Webhook": true,
```

```suggestion
            "v2Webhook": false,
```

Set both v2 and v2webhook to false by default. It also breaks when v2 is enabled but v2webhook is disabled so maybe these should just be 1 option? I don't really see the reason for having them split up.